oca_port/cli/main.py

```diff
@@ -33,6 +33,10 @@
 
     $ oca-port 14.0 16.0 shopfloor --upstream-org camptocamp
 
+To rename a module during its migration (or compare commits of a renamed module):
+
+    $ oca-port origin/16.0 origin/18.0 stock_packaging_calculator product_packaging_calculator
```

Comment:
```suggestion
    $ oca-port origin/16.0 origin/18.0 stock_packaging_calculator --rename product_packaging_calculator
```
I would make it more explicit